dev/WebView2/WebView2.h

```diff
@@ -48,7 +48,7 @@ class CoreWebView2InitializedEventArgs :
 };
 
 template <typename D, typename T, typename ... I>
-struct __declspec(empty_bases) DeriveFromContentControlHelper_base : winrt::Windows::UI::Xaml::Controls::ContentControlT<D, winrt::default_interface<T>, winrt::composable, I...>
+struct __declspec(empty_bases)DeriveFromContentControlHelper_base : winrt::Windows::UI::Xaml::Controls::ContentControlT<D, winrt::default_interface<T>, winrt::composable, I...>
```

Comment:
Is this intended (and does it make a difference)?

Reply:
VisualStudio's Format Document feature removed this whitespace.